package.json

```diff
@@ -13,15 +13,17 @@
     "start": "nodemon server",
     "build": "webpack",
     "watch": "webpack -w",
-    "test": "jest"
+    "test": "jest",
+    "sass": "sass --watch public/styles.scss public/styles.css"
```

Comment:
@jimmyko360 I noticed you've added a new command to watch the scss/css files.  From a quick web search it seems possible to have webpack do this work.  Happy to approve this for the time being but I think it would be great to move the sass compiling in to webpack so we only have to use one build tool.  What do you think?

This was the site I found that mentions compiling scss with webpack: https://blog.jscrambler.com/easy-custom-webpack-setup-for-react-js-applications/